apps/flame-defi/app/route-animation.tsx

```diff
@@ -0,0 +1,19 @@
+"use client";
```

Comment:
Let's put this in `components` or _maybe_ `features` to avoid putting things directly in project root.

Edit: looking at it more, i'd be okay with just defining the component in the layout file

Reply:
If we define the RouteAnimation in the main layout.tsx file we'll have to add "use-client" and it won't be a server component anymore

Reply:
Oh that makes sense! Can we put in `components/` then?